app/installer.sh

```diff
@@ -105,12 +105,12 @@ main() {
 
   if [ "$VERSION" = 'preview' ]; then
 
-    if ! command -v gh >/dev/null 2>&1; then
-      err "Must have github's gh CLI installed to install a preview version."
+    if ! command -v jq >/dev/null 2>&1; then
+      err "Must have jq installed to install a preview version."
     fi
 
     log "Fetching latest preview commit.."
-    commit=$(gh run list --repo "${REPO}" --branch "${BRANCH}" --workflow build-test-distribute -L 200 --json event,headSha,conclusion --jq '[.[] | select(.conclusion == "success" and .event == "push")] | first | .headSha[0:9]')
+    commit=$(curl -s --request GET --url "https://api.cloudsmith.io/v1/packages/kong/${PRODUCT_NAME}-binaries-preview/?page=1&page_size=1&query=filename%3A0.0.0-preview&sort=-date" --header 'accept: application/json' | jq -r '.[0].version')
```

Comment:
`curl https://deploy-preview-1733--kuma.netlify.app/installer.sh | PRODUCT_NAME=kuma VERSION=preview sh -`

Works but `curl https://deploy-preview-1733--kuma.netlify.app/installer.sh | VERSION=preview sh -` fails with:

```
curl https://deploy-preview-1733--kuma.netlify.app/installer.sh | VERSION=preview sh -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6571  100  6571    0     0  21904      0 --:--:-- --:--:-- --:--:-- 21903
INFO    Welcome to the Kuma automated download!
INFO    Fetching latest preview commit..
jq: error (at <stdin>:0): Cannot index object with number
```

Reply:
Product name is `Kuma` vs `kuma` this likely explains the issue

Reply:
fixed